src/code/z_demo.c

```diff
@@ -46,7 +46,7 @@ u32 D_8011E304[] = { 0x0200B650, 0x02015600, 0x02014F80, 0x02003F80, 0x02000330,
 u16 D_8015FCC0;
 u16 D_8015FCC2;
 u16 D_8015FCC4;
-s16 D_8015FCC6;
+s16 sActiveCamId;
```

Comment:
Yeah true. Thoughts on `sPrevCamId`?

Reply:
Maybe something indicating it's the "gameplay" camera at that point, like `sGameplayCamId`, as opposed to the camera(s) used for cutscene purposes in this context? I think this could make the most sense since the system is also responsible for bringing the camera of this id back to be active once the cutscene is finished. Also it could fit with the `Gameplay_CopyCamera` call used at the end of some cutscenes (see `func_80068DC0`), which I'm guessing is meant to set the gameplay camera to the cutscene one at the end for a smoother transition.

Otherwise we could go with a more generic approach like "prev" or "save", but it might not be very clear, so I'm not sure.

Reply:
I agree "active" is misleading
`sPrevCamId` sounds fine
or maybe `sPrevActiveCamId` / `sReturnToCamId` if you like that

Reply:
Apologies for the 4-month delay in responding, I haven't had the motivation for this particular PR but I'm getting back into it now :)

I personally like `sReturnToCamId` the most out of the suggestions, so I'll change it to that for now. But I'll keep this thread open for further discussion.

I'm not sure if I like the term "Gameplay" as it's not really used anywhere else to distinguish between the 4 camera's, and `Gameplay_CopyCamera` just leads to `Camera_Copy` anyway, and it just an extra layer to check for active camera id's.

Reply:
Yeah returnToCam seems fine as well. It's unusual with the rest of the codebase right now, but it should work